substrate/frame/support/src/storage/mod.rs

```diff
@@ -1400,8 +1421,7 @@ pub trait StoragePrefixedMap<Value: FullCodec> {
 /// This trait is sealed.
 pub trait StorageAppend<Item: Encode>: private::Sealed {}
 
-/// Marker trait that will be implemented for types that support to decode their length in an
```

Comment:
idk why these lines are removed?

Reply:
I thought to remove them after this feedback PR comment https://github.com/paritytech/polkadot-sdk/pull/1932#discussion_r1365717982